components/Sections/Introduction/Content.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import Image from "next/image";
+import { Link as ScrollLink } from "react-scroll";
+
+import Button from "components/Button";
+import { ButtonVariant } from "components/Button/types";
+import { scrollDuration } from "constants/scroll";
+
+import { IntroductionSectionContentProps } from "./types";
+import { introductionSectionId, introductionSectionContent } from "./constants";
+
+const Content: React.FC<IntroductionSectionContentProps> = ({
+  backgroundSrc = introductionSectionContent.backgroundSrc,
+  buttonText = introductionSectionContent.buttonText,
+  title = introductionSectionContent.title,
+}) => (
+  <section id={introductionSectionId} className="flex items-center py-10 px-8 md:p-20 relative h-section-introduction bg-white">
+    <Image
+      src={backgroundSrc}
+      className="absolute object-cover object-center h-full w-full"
+      layout="fill"
+      quality={100}
+      loading="eager"
+    />
+
+    <div className="max-w-screen-lg w-full mx-auto absolute right-0 left-0 px-5">
```

Comment:
Is it necessary to use absolute positioning here too?

Reply:
Fixed on [`8883bf7`](https://github.com/LauraBeatris/floripamais/pull/15/commits/8883bf780995612494f27733f94d75ffe4b07f8d)

The background image has a position ``absolute`` and so has a higher z-index - That's why we also need to add a z-index value to the section content.